it/builders/src/test/java/com/linecorp/armeria/OverriddenBuilderMethodsReturnTypeTest.java

```diff
@@ -42,7 +42,7 @@ class OverriddenBuilderMethodsReturnTypeTest {
 
     @Test
     void methodChaining() {
-        final String packageName = "com.linecorp.armeria";
+        final String packageName = "com.linecorp.armeria.server";
```

Comment:
Is there any reason for changing this value?

Reply:
It's my fault 😱